### PR TITLE
New registration relabel operator name on check answers page

### PR DESCRIPTION
**Pull request by Beckyrose200**
https://eaflood.atlassian.net/browse/RUBY-1873
This is a label change for the check your answers page. When the business type is either a LLP or company the label is changed to Registered Name (as this is pulled from companies house and isn't entered by the user) however if its anything else i.e. charity then the label is Operator Name. 

Spec file has been changed to test the new method and to test a charity on the 'check your answers page' as well.

### CODE REVIEW

File: app/models/concerns/waste_exemptions_engine/can_use_new_registration_workflow.rb [hunk 1]
```diff
@@ -322,6 +322,10 @@ module CanUseNewRegistrationWorkflow
                       to: :check_registered_name_and_address_form,
                       unless: :skip_registration_number?
 
+          transitions from: :operator_postcode_form,
```
Comment (PaulDoyle-DEFRA) [inline app/models/concerns/waste_exemptions_engine/can_use_new_registration_workflow.rb:325]: This is unrelated to the change as described in the PR description. Was it included accidentally?

Comment (Beckyrose200) [reply app/models/concerns/waste_exemptions_engine/can_use_new_registration_workflow.rb:325]: Its been duplicated by the looks of it, I rebased the branch which had quite a lot of changes and I think its duplicated itself when I was accepting the current changes. Will remove this now :)

**Review by PaulDoyle-DEFRA — APPROVED** [review]
